cmd/sonobuoy/app/gen_plugin_def.go

```diff
@@ -186,6 +195,18 @@ func genPluginDef(cfg *GenPluginDefConfig) ([]byte, error) {
 		cfg.def.PodSpec.NodeSelector = cfg.nodeSelector
 	}
 
+	if len(cfg.configMapFiles) > 0 {
+		cfg.def.ConfigMap = map[string]string{}
+	}
+	for _, v := range cfg.configMapFiles {
+		fData, err := os.ReadFile(v)
+		if err != nil {
+			return nil, errors.Wrapf(err, "failed to read file %q", v)
+		}
+		base := filepath.Base(v)
+		cfg.def.ConfigMap[base] = string(fData)
```

Comment:
Possibility of overwriting file content if `base` happens to be same name from multiple locations.

Reply:
True but wouldnt that be the case for all configmaps? This is for a particular plugin and each plugins configmaps are put into separate objects, so it shouldn't conflict between plugins.